modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx

```diff
@@ -411,8 +431,10 @@ const FrontendDataSetContent = ({
 					},
 					[]
 				),
-				onLoad: (bindingContexts: any) => {
-					bindingContexts.forEach(
+				onLoad: (
+					resolutions: Array<ClientExtensionResolution<any>>
+				) => {
+					resolutions.forEach(
```

Comment:
Do we need to signal errors in resolutions here? `CellRenderer` component seems to blindly trust in the resolved `htmlElementBuilder` with no sensible default in case it's not loaded.